keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -32,17 +33,20 @@
     - [Admission Controller](#admission-controller)
     - [Eviction Manager](#eviction-manager)
     - [Pod Overhead](#pod-overhead)
+    - [Hugepages](#hugepages)
+    - [Memory Manager](#memory-manager)
+    - [In-Place Pod Resize](#in-place-pod-resize)
+      - [API changes](#api-changes)
+      - [Resize Restart Policy](#resize-restart-policy)
+      - [Implementation Details](#implementation-details)
+    - [[Scopred for Beta] CPU Manager](#scopred-for-beta-cpu-manager)
+    - [[Scoped for Beta] Topology Manager](#scoped-for-beta-topology-manager)
     - [[Scoped for Beta] User Experience Survey](#scoped-for-beta-user-experience-survey)
     - [[Scoped for Beta] Surfacing Pod Resource Requirements](#scoped-for-beta-surfacing-pod-resource-requirements)
       - [The Challenge of Determining Effective Pod Resource Requirements](#the-challenge-of-determining-effective-pod-resource-requirements)
       - [Goals of surfacing Pod Resource Requirements](#goals-of-surfacing-pod-resource-requirements)
-      - [Implementation Details](#implementation-details)
+      - [Implementation Details](#implementation-details-1)
```

Comment:
nit: why implementation-details-1?

Reply:
Because there's another "implementation details" on line 41. This is just how github generates the anchors.